src/std/rc.md

```diff
@@ -25,8 +25,11 @@ context.
 
 <details>
 
+* `Rc`'s Count ensures that its contained value is valid for as long as there are references.
 * Like C++'s `std::shared_ptr`.
-* `clone` is cheap: creates a pointer to the same allocation and increases the reference count.
+* `clone` is cheap: it creates a pointer to the same allocation and increases the reference count. Does not make a deep clone and can generally be ignored when looking for performance issues in code.
 * `make_mut` actually clones the inner value if necessary ("clone-on-write") and returns a mutable reference.
+* Use `Rc::strong_count` to check the reference count.
+* Compare the different datatypes mentioned. `Box` enables (im)mutable borrows at compile time, `RefCell` enables (im)mutable borrows at run time and will panic if it fails at runtime.
```

Comment:
I'm not sure quite what you mean by this. `RefCell` enables (single-threaded) interior mutability through dynamically checked borrow rules.

Reply:
Hi, I can rewrite a little in case it's confusing. "With references and Box<T>, the borrowing rules’ invariants are enforced at compile time. With RefCell<T>, these invariants are enforced at runtime. " [book](https://doc.rust-lang.org/book/ch15-05-interior-mutability.html)